README.adoc

```diff
@@ -0,0 +1,193 @@
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+// Links (alphabetical order)
+:apache_license: http://www.apache.org/licenses/LICENSE-2.0[Apache License, Version 2.0]
+:application_monitoring_operator: https://github.com/integr8ly/application-monitoring-operator[application-monitoring-operator]
+:export_policy: https://aerogear.org/legal/export.html[AeroGear Export Policy]
+:aerogear_freenode: irc://irc.freenode.net/aerogear[#aerogear on FreeNode IRC]
+:aerogear_jira: https://issues.jboss.org/projects/AEROGEAR/issues[AeroGear on JBoss Jira]
+:aerogear_matrix: https://matrix.to/#/!IipcvbGVqkiTUQauSC:matrix.org[#aerogear:matrix.org on Matrix]
+:mailing_list: https://groups.google.com/forum/#!forum/aerogear[Google Groups Mailing List]
+:minishift: https://github.com/minishift/minishift[Minishift]
+:rh_product_security: https://access.redhat.com/security/team/contact[Red Hat Product Security team]
+:minishift_cors: https://github.com/aerogear/mobile-developer-console#enable-cors-in-the-openshift-cluster[steps]
+
+= Mobile Developer Console Operator
+
+ifdef::status[]
+.*Project health*
+image:https://travis-ci.com/aerogear/mobile-developer-console-operator.svg?branch=master[Build Status (Travis), link=https://travis-ci.com/aerogear/mobile-developer-console-operator.svg?branch=master]
+image:https://img.shields.io/:license-Apache2-blue.svg[License (License), link=http://www.apache.org/licenses/LICENSE-2.0]
+endif::[]
+
+
+The Mobile Developer Console Operator for Kubernetes provides an easy way to
+install and manage an AeroGear Mobile Developer Console Operator on Kubernetes.
+
+toc::[]
+
+
+== Limitations
+
+// https://issues.jboss.org/browse/AEROGEAR-9162
+[NOTE]
+====
+This operator currently only works on OpenShift. This is because it
+provisions `Route`s and also relies on the fact that OpenShift
+provides an OAuth server. In future we aim to make it work on vanilla
+Kubernetes also.
+====
+
+== Minishift Pre-requisites
+
+. Enable CORS addon on Minishift
+
+Follow these {minishift_cors} in order to enable CORS on Minishift.
+
+....
+ if minishift addons list | grep cors ; then	
+    minishift addons apply cors	
+else	
+    MINISHIFT_ADDONS_PATH=/tmp/minishift-addons	
+    rm -rf $MINISHIFT_ADDONS_PATH && git clone https://github.com/minishift/minishift-addons.git $MINISHIFT_ADDONS_PATH	
+    # Not needed after https://github.com/minishift/minishift-addons/pull/187 is merged	
+    cd $MINISHIFT_ADDONS_PATH	
+    git fetch origin pull/187/head:cors-fix && git checkout cors-fix	
+    minishift addons install /tmp/minishift-addons/add-ons/cors	
+    minishift addons apply cors	
+fi	
+minishift addon apply cors
+....
+
+. Export required OPENSHIFT_HOST variable
+....
+export OPENSHIFT_HOST=$(minishift ip)
+....
+
+== Installation
+
+As a user with admin permissions, you can easily install the
+mobile-security-service-operator in your OpenShift cluster as follows:
+
+....
+make cluster/prepare
+make install
+....
+
+
+== TODO
```

Comment:
@aliok I added the new section from the md README here as a TODO so that we know we should remove it when the steps are done. Could you give it a quick eye before I merge it. I didn't change it, just copy and pasted and added a line to tell the user to take the steps for now.

Reply:
Sure, sounds good to me.